### ISSUE TITLE
DB/Quest: Proof Of Treachery appears broken

### ISSUE THREAD

**Issue by TQPS** [TrinityCore/TrinityCore#27011]
### Description

After completing [Survey Alcaz Island](https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?quest=11142), in this timeline (3.0.2-4.0.3a), Lady Jaina offers the next quest [Proof Of Treachery](https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?quest=11222) and is supposed to offer a port to Stormwind to talk to the King.

- Neither the quest nor the portal is being offered. 
- The quest page in AOWOW doesn't list Lady Jaina as a start NPC 
- The quest completion portion has remains of the previous quest version with [Bolvar](https://wowpedia.fandom.com/wiki/Warn_Bolvar!) (*which is outdated*)

Quest history - [here](https://wowpedia.fandom.com/wiki/Proof_of_Treachery)
The one of interest to us is at the bottom of the page for v3.0.2

### Expected behaviour

Jaina should offer Proof of Treachery and content should be reflective of current timeline (as of WOTLK).

### Steps to reproduce the problem

1. Undertake questline beginning with [A Disturbing Development](https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?quest=11136)
or
Just give yourself [Survey Alcaz Island](https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?quest=11142) and complete it
2. Jaina wont offer the followup [Proof Of Treachery](https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?quest=11222)

<details>

Survey Alcaz Island finished but it is not followed up with Proof of Treachery

![image](https://user-images.githubusercontent.com/66294837/135735069-7c585617-46a3-4fad-a4d5-b60ce4d1f5a9.png)

</details>

### Branch

3.3.5

### TC rev. hash/commit

TrinityCore rev. 3630040 2021-09-23 22:10:41 +0300 (3.3.5 branch) (Win64, RelWithDebInfo, Static)

### Operating system

Windows 7 x64

### Custom changes

Trickerer's Bots

**Comment by Aokromes** [comment]
https://www.wowhead.com/quest=11222/deprecated-proof-of-treachery#comments:id=407410

**Comment by TQPS** [comment]
Then this doesn't make sense

![image](https://user-images.githubusercontent.com/66294837/135736094-aea2366b-f688-47a0-b8a6-dd32f5b79761.png)

Taken from https://wowpedia.fandom.com/wiki/Proof_of_Treachery

Nor this ....

https://www.youtube.com/watch?v=AdlcugU8tJM

**Comment by TQPS** [comment]
https://www.wowhead.com/quest=27240/proof-of-treachery

The quest never went away, it kept changing during timelines

**Comment by Aokromes** [comment]
a second quest with that name was re-added on 4.0.3

**Comment by TQPS** [comment]
The quest line in TC's current state is not finished - it's missing last 2 quests.

I'll fix it up my end. Letting this go.

**Comment by offl** [comment]
Looks like first they updated texts of https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?quest=11222, then they removed it (https://www.wowhead.com/quest=11222/deprecated-proof-of-treachery#comments:id=407410:reply=358279)

In cata they added it back as https://www.wowhead.com/quest=27240/proof-of-treachery

**Comment by TQPS** [comment]
This is the TBC version of the quest line:
https://tbc.wowhead.com/quest=11222/warn-bolvar

Then WOTLK came out and as it says at the bottom, Proof of Treachery REPLACED Warn Bolvar
![image](https://user-images.githubusercontent.com/66294837/135771513-01291aab-1c5d-46ab-8d05-4956195c5a9e.png)

And again, the quest was changed in CATA, Legion and BFA.

So if you look at CATA DB:

```sql
SELECT id,RewardNextQuest,LogTitle,LogDescription,QuestDescription FROM quest_template WHERE `LogTitle` LIKE '%Proof of Treachery%';
```
quest_template
---
| id | RewardNextQuest | LogTitle | LogDescription | QuestDescription | 
| ---: | ---: | --- | --- | --- | 
| 11222 | 11223 | [DEPRECATED]Proof of Treachery | Speak to King Varian Wrynn in Stormwind Keep. | Although Onyxia has been exposed and removed from the court, the Defias who carried out her plan must be punished. Carry word to King Varian Wrynn in Stormwind castle that the Defias are the ones who carried out his kidnapping. I have little doubt that the king will mete out swift and terrible justice to those involved.$B$BI can use my magic to expedite your trip to Stormwind. Just speak to me when you're prepared to go, $N. | 
| 27240 | 27241 | Proof of Treachery | Speak with Jaina to travel to Stormwind and meet with King Varian Wrynn. | Although Onyxia has been exposed and removed from the court, the Defias who carried out her plan must be punished. Carry word to King Varian Wrynn in Stormwind castle that the Defias are the ones who carried out his kidnapping. I have little doubt that the king will mete out swift and terrible justice to those involved.$B$BI can use my magic to expedite your trip to Stormwind. Just speak to me when you're prepared to go, $N. | 

27240 is the CATA replacement - which means 11222 is the Wrath version.

This is the quest line that introduced the free ports from/to Theramore/Stormwind. Back in the days of L80 Ony raid nights guild mates would use either Proof of Treachery or its followup, Return to Jaina to get free ports. I'm a completionist so I already had completed it, but as a mage, I didn't need them. But if you played WOTLK chances are you also knew people who used this method for free rides.

Without fixing this quest line, the last 2 quests wont be added until WOTLK Classic when I am sure someone will pick up on it.

**Comment by offl** [comment]
I understand what you're trying to say but look at all comments of those 2 quests
And https://www.wowhead.com/quest=11142/deprecated-survey-alcaz-island#comments:id=408542

And especially at last link
https://web.archive.org/web/20090816150014/http://wow.allakhazam.com/db/quest.html?wquest=11223
https://web.archive.org/web/20101007022203/http://wow.allakhazam.com/db/quest.html?wquest=11222#Comments

And the reason why they changed text and name of quest is probably because it still was in player's quest logs

Or you want to make sure players who haven't turned in those quests can still use portals? The only way to implement that is make sure quest is in your quest log, then block possibility to re-accept that quest or accept next quest in chain
So it's not possible
In other words for your character on TC it works like for all new characters on retail, next quest simply cannot be accepted but can be turned in if it still in your quest log

**Comment by TQPS** [comment]
It's ok. Leave it be.

I've already added the queststarter and enders to my DB.
